cloud/pkg/admissioncontroller/config/kube.go

```diff
@@ -0,0 +1,27 @@
+package config
+
+import (
+	"github.com/kubeedge/beehive/pkg/common/config"
+	"github.com/kubeedge/beehive/pkg/common/log"
+)
+
+// Kube container Kubernetes related configuration
+var Kube *KubeInfo
+
+func init() {
+	Kube = NewKubeInfo()
+
+	if km, err := config.CONFIG.GetValue("controller.kube.master").ToString(); err != nil {
+		log.LOGGER.Errorf("Controller kube master not set")
+	} else {
+		Kube.KubeMaster = km
+	}
+	log.LOGGER.Infof("Controller kube master: %s", Kube.KubeMaster)
+
+	if kc, err := config.CONFIG.GetValue("controller.kube.kubeconfig").ToString(); err != nil {
+		log.LOGGER.Errorf("Controller kube config not set")
+	} else {
+		Kube.KubeConfig = kc
+	}
+	log.LOGGER.Infof("Controller kube config: %s", Kube.KubeConfig)
```

Comment:
can we also fetch kubenamespace from devicecontroller configuration ?

Reply:
This namespace no need to match with how devicecontroller's namespace is set, I am going to isolate all those into a separate namespace in the future.